### ISSUE TITLE
Patronage bug

### ISSUE THREAD

**Issue by AdityaMH** [yairm210/Unciv#4181]
Every time i choose patronage, always going to crash instantly. And my save game data lead to corrupted.
(Maybe because from rekMod)

About save data: sorry, i can't upload to github because github system bug. Maybe i will upload to discord.

**Comment by xlenstra** [comment]
Patronage isn't in the main game (yet), so this is a mod issue

**Comment by ravignir** [comment]
give me more info, which policy causes the issue?

**Comment by AdityaMH** [comment]
Just choose patronage policy, then end turn. Instantly crashed and sometimes my save data lead to corrupted.

Not all crashed problem cause by mod is from mod itself. But because missing or removed some unciv base data on lastest version.
Just like #3918



Before version 3.15.0, never cause problem, run normally.

**Comment by AdityaMH** [comment]
That's weird, everytime i choose patronage always lead to crashed. But sometimes i can resume the game, sometime my save data corrupted.

**Comment by ravignir** [comment]
`if (!ruleSet.policies.containsKey(policy) && !(policy == "Entrepreneurship" || policy == "Patronage")) civinfo.policies.adoptedPolicies.remove(policy)`
I believe this causes the crash.

**Comment by xlenstra** [comment]
I think this part may also play a role (from in the policymanager class)
```
            if (adoptedPolicies.contains("Patronage")) {
                adoptedPolicies.add("Merchant Navy")
                adoptedPolicies.remove("Patronage")
            }
            if (adoptedPolicies.contains("Entrepreneurship")) {
                adoptedPolicies.add("Naval Tradition")
                adoptedPolicies.remove("Entrepreneurship")
            }
```

**Comment by ravignir** [comment]
yes, you are right, the one i postedjust "unpicks" the policy as soon as it is adopted, the one you posted tries to replace it with a policy that does not exist.

**Comment by ravignir** [comment]
close?
